cfg/cis-1.9/policies.yaml

```diff
@@ -0,0 +1,398 @@
+              set: true
+        remediation: |
+          Where possible replace any use of wildcards ["*"] in roles with specific
+          objects or actions.
+          Condition: is_compliant is false if ["*"] is found in rules.
+          Parent: 5.1.3.
+        scored: true
+      - id: 5.1.3.2
+        text: "Minimize wildcard use in ClusterRoles (Automated)"
+        audit: |
+          kubectl get clusterroles -o custom-columns=CLUSTERROLE_NAME:.metadata.name --no-headers | while read -r clusterrole_name
+          do
+            clusterrole_rules=$(kubectl get clusterrole "${clusterrole_name}" -o=json | jq -c '.rules')
+            if echo "${clusterrole_rules}" | grep -q "\[\"\*\"\]"; then
+              is_compliant="false"
+            else
+              is_compliant="true"
+            fi;
+          echo "**clusterrole_name: ${clusterrole_name} clusterrole_rules: ${clusterrole_rules} is_compliant: ${is_compliant}"
+          done
+        use_multiple_values: true
+        tests:
+          test_items:
+            - flag: "is_compliant"
+              compare:
+                op: eq
+                value: true
+              set: true
+        remediation: |
+          Where possible replace any use of wildcards ["*"] in clusterroles with specific
+          objects or actions.
+          Condition: is_compliant is false if ["*"] is found in rules.
+          Parent: 5.1.3.
+        scored: true
+      - id: 5.1.4
+        text: "Minimize access to create pods (Automated)"
+        audit: |
+          echo "canCreatePodsAsSystemAuthenticated: $(kubectl auth can-i create pods --all-namespaces --as=system:authenticated)"
+        tests:
+          test_items:
+            - flag: "canCreatePodsAsSystemAuthenticated"
+              compare:
+                op: eq
+                value: no
+        remediation: |
+          Where possible, remove create access to pod objects in the cluster.
+        scored: true
+      - id: 5.1.5
+        text: "Ensure that default service accounts are not actively used. (Automated)"
+        audit: |
+          kubectl get serviceaccount --all-namespaces --field-selector metadata.name=default -o=json | jq -r '.items[] | " namespace: \(.metadata.namespace), kind: \(.kind), name: \(.metadata.name), automountServiceAccountToken: \(.automountServiceAccountToken | if . == null then "notset" else . end )"' | xargs -L 1
+        use_multiple_values: true
+        tests:
+          test_items:
+            - flag: "automountServiceAccountToken"
+              compare:
+                op: eq
+                value: false
+              set: true
+        remediation: |
+          Create explicit service accounts wherever a Kubernetes workload requires specific access
+          to the Kubernetes API server.
+          Modify the configuration of each default service account to include this value
+          `automountServiceAccountToken: false`.
+        scored: true
+      - id: 5.1.6.1
```

Comment:
```suggestion
      - id: 5.1.6
```

Reply:
This is the same approach I've used for 5.1.3, since 5.1.6 was complex to satisfy with only one check so I used the Artifacts as baseline.

<img width="885" alt="image" src="https://github.com/aquasecurity/kube-bench/assets/12878731/86c818fb-7eab-445c-b5b6-7b498acba4f4">